recipe/build.sh

```diff
@@ -73,4 +71,4 @@ fi
 cd python
 export LIBSHARP="${PREFIX}"
 export LDSHARED="${CC} -shared"
-${PYTHON} setup.py install --prefix="${PREFIX}"
+"${PYTHON}" -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
Are these quotes right?

Reply:
That was copied from the h5py feedstock, but I'll remove them.  Should perhaps put an eval at the front of the command as well...